### ISSUE TITLE
Make sure tiles update when animating

### ISSUE THREAD

**Issue by djhoese** [ssec/sift#27]
In GitLab by @djhoese on Sep 30, 2015, 11:03

The pieces aren't connected in the way they will be, but my current testing version does not update tile data while an animation is happening. So this is less of a current bug and more of a "make sure this works when things are more finalized".

**Comment by djhoese** [comment]
In GitLab by @rayg-ssec on Sep 30, 2015, 17:40

Similarly, zooming and panning should not slow down or stop animation. Not sure if this is a separate ticket or not. Can wait until after demo.

**Comment by djhoese** [comment]
In GitLab by @djhoese on Oct 8, 2015, 08:53

This has been fixed. The main issue was that retiling was being "scheduled" or assessed for every draw event. So while animating, every new frame triggered a redraw which in turn started the debouncing timer for reassessment. So on a quick animation timer the debouncing timer was restarted every frame causing a timeout to never occur. If the animation timer was slowed down to be longer than the debounce timer everything worked as expected.

To fix this I discovered a better signal to connect to `self.scene_manager.main_canvas.transforms.changed` and used it. This signal/event is emitted every time the transform of the canvas is changed. This happens on pan, zoom, and resize which is what was originally desired for the retiling debounce in the first place. This fix also means that retilings are never scheduled during animation unless a pan/zoom/resize occurs.
